packages/kbn-unified-field-list/src/components/field_categorize_button/categorize_trigger_utils.ts

```diff
@@ -6,11 +6,8 @@
  * Side Public License, v 1.
  */
 
-import {
-  type UiActionsStart,
-  CATEGORIZE_FIELD_TRIGGER,
-  CategorizeFieldContext,
-} from '@kbn/ui-actions-plugin/public';
+import type { UiActionsStart, CategorizeFieldContext } from '@kbn/ui-actions-plugin/public';
+import { CATEGORIZE_FIELD_TRIGGER } from '@kbn/ui-actions-browser/src/triggers';
```

Comment:
in the next file it's imported like this, so this could be aligned
```suggestion
import { CATEGORIZE_FIELD_TRIGGER } from '@kbn/ui-actions-browser';
```